docs/guides/build-with-magicavoxel.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Build with MagicaVoxel
+type: guides
+layout: docs
+parent_section: guides
+order: 3
+---
+
+[magicavoxel]: https://ephtracy.github.io/
+
+[MagicaVoxel][magicavoxel] is a free and user-friendly tool that lets you build
```

Comment:
They should live here because people need discoverable guides on how to use A-Frame practically. Having them in the documentation ensures discoverability, versioning, and the ability to use relative links.